content/articles/ordering-a-wildcard-certificate.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Ordering a Wildcard SSL Certificate
+excerpt: How to order a wildcard SSL certificate with DNSimple.
+categories:
+- SSL Certificates
+---
+
+# Ordering a Standard SSL Certificate
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Wildcard certificates allow you to secure an unlimited number of names belonging to a single subdomain level. Wildcard certificates can make certificate management easier in some cases. However, you should avoid them unless you have a specific need, such as multi-tenant applications.
+
+Wildcard certificates are no different from single name certificates. They contain an `*` as the hostname.
+
```

Comment:
Could you put an example there: 
<img width="464" alt="wildcard" src="https://user-images.githubusercontent.com/32067948/49108292-d4bb8080-f255-11e8-8b4f-177bbd1d4250.png">
I found this image on a blog and I feel that really help to understand.  So not the same of course but something similar?

Reply:
![screenshot 2018-11-28 at 09 17 00](https://user-images.githubusercontent.com/584259/49138022-624aaf00-f2ee-11e8-8baa-fc99a17668be.png)

I added this ASCII chart to explain the concept.